### PR TITLE
Display expiry date on Preview and Detail View for a segment of a TV/radio transcript

### PR DESCRIPTION
**Pull request by dzonidoo**
CPCN-30

## Checklist

- [x] This pull request is not adding new forms that use redux
- [x] This pull request is adding missing TypeScript types to modified code segments where it's easy to do so with confidence
- [x] This pull request is replacing `lodash.get` with optional chaining for modified code segments

### CODE REVIEW

File: assets/wire/components/fields/ExpiryDateLabel.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import PropTypes from 'prop-types';
+import {gettext} from 'utils';
+import {get} from 'lodash';
+
+export function ExpiryDateLabel ({item, filterGroupLabels}: any) {
+    if (item.expiry == null) {
+        return null;
+    }
+
+    const label = get(filterGroupLabels, 'expiry', gettext('Expiry Date'));
```
Comment (petrjasek) [inline assets/wire/components/fields/ExpiryDateLabel.tsx:11]: no get pls

File: assets/wire/components/fields/ExpiryDateLabel.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    const value = item.expiry;
+    const text = label + ': ' + value;
```
Comment (petrjasek) [inline assets/wire/components/fields/ExpiryDateLabel.tsx:13]: the value should be formatted using `utils.formatDate`

Comment (dzonidoo) [reply assets/wire/components/fields/ExpiryDateLabel.tsx:13]: can u give me more informations? :D

Comment (petrjasek) [reply assets/wire/components/fields/ExpiryDateLabel.tsx:13]: we don't want to display in the UI the server format `2023-12-07T09:53:00` , we have a function for that in utils:
```ts
import {formatDate} from 'utils';
value = formatDate(item.expiry);
```

File: assets/wire/components/fields/ExpiryDateLabel.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {gettext, formatDate} from 'utils';
+
+export function ExpiryDateLabel ({item, filterGroupLabels}: any) {
```
Comment (MarkLark86) [inline assets/wire/components/fields/ExpiryDateLabel.tsx:5]: @dzonidoo Please add types as per the PR checklist. This includes adding `expiry` to the `IArticle` (https://github.com/superdesk/newsroom-core/blob/develop/assets/interfaces/content.ts#L13), and avoiding use of `any` and `PropType` and instead use the appropriate `IProps` interface to be used for this component

**Review by MarkLark86 — APPROVED** [review]

**Pull request merged**